### PR TITLE
Disable consensus and close p2p service.

### LINKED ISSUE #3: Disable the consensus module in the miner

### LINKED ISSUE #6: Disable P2P component
As the sequencer will start out as a centralized server, the P2P component will not be needed and can be stripped/disabled.

### PR DESCRIPTION
**Pull request by mask-pp**

### CODE REVIEW

**Review by jules — APPROVED** [review]
LGTM!

File: eth/backend.go [hunk 1]
```diff
@@ -538,7 +538,7 @@ func (s *Ethereum) Start() error {
 		maxPeers -= s.config.LightPeers
 	}
 	// Start the networking layer and the light server if requested
-	s.handler.Start(maxPeers)
+	//s.handler.Start(maxPeers)
```
Comment (icemelon) [inline eth/backend.go:541]: Comment out from L533 to L541.

Comment (mask-pp) [reply eth/backend.go:541]: Code line from L533 to L539 is useless, close that is mutch better.

**Review by icemelon — APPROVED** [review]
LGTM

**Pull request merged**